### DISCUSSION TITLE
.local ?

### DISCUSSION THREAD

**Discussion by thesebastianf** [AdguardTeam/AdGuardHome#4636] [Q&A]
Hey
when my Router was set aß DNS entry for my clients I could use Just the Host names to reach my devices Like e.g.

myhomeserver   was resolved fine for a lan device

I added in the Adguard settings my Router again as local DNS resolver

However I must call now

myhomeserver.local to get to the LAN device

I know I can also add every device in DNS rewrites.

But is there also a solution that avoids the .local and all Hosts can bei reached again by solely the Hostname as Standard?

Thank you.

**Comment by ainar-g** [comment]
You can point all `.local` requests to your router's DNS service by using the following upstream record:

```
[/local/]192.168.123.45
```

**Reply by thesebastianf** [reply]
Mhh
My Goal is to avoid a .local request.

Before Adguard If I called
"myhomeserver" it was resolved correct to a 192.168.x.x

I want to avoid to call with a "myhomeserver.local"

Nethertheless I might have understood your advice wrong and Just tried to add your String to upstream.
But it did Not do the Trick for me.

**Reply by ainar-g** [reply]
@thesebastianf, you can send all requests to unqualified hosts (that is, those without a dot) by using the following notation:

```none
[//]192.168.123.45
```

Where `192.168.123.45` is the address of your router.

**Reply by thesebastianf** [reply]
Unfortunately still
DNS_PROBE_FINISHED_NXDOMAIN

while a .local opens suceeds again

**Reply by ainar-g** [reply]
@thesebastianf, are you using AGH as the DHCP server as well?

**Reply by thesebastianf** [reply]
No the Router is the DHCP Server and announces AGH as DNS to the clients

**Reply by ainar-g** [reply]
Well, that's weird.  If the router used to be the DNS server, and it resolved names such as `myhomeserver` normally, but that doesn't work through AdGuard Home's upstream for domain feature then perhaps the machine that you use to make those queries automatically adds a local suffix.  It's hard to tell without knowing all the details, but I'd suggest looking through the query log to see if you can find any queries with interesting suffixes.
